README.md

```diff
@@ -29,66 +16,73 @@ npm i git-guppy --save-dev
 
 ## Usage
 
-### Git integration
+### 1. Declaring hooks as gulp tasks
```

Comment:
**Motivation**: Organized into steps to make it easier for users to locate themselves in the explanation.